cf/dimensioncoordinate.py

```diff
@@ -339,155 +339,223 @@ def create_bounds(
-                        12:00`` setting, ``cellsize=cf.M()`` will result in
-                        bounds of ``(1984-12-01, 1985-01-01), (1985-01-01,
-                        1985-02-01)`` (see `cf.M` for details).
-
-                    *Parameter example:*
-                        For coordinates ``1984-12-01 12:00, 1985-01-01
-                        12:00`` setting, ``cellsize=cf.M(day=20)`` will
-                        result in bounds of ``(1984-11-20, 1984-12-20),
-                        (1984-12-20, 1985-01-20)`` (see `cf.M` for details).
-
-                    *Parameter example:*
-                        For coordinates ``1984-03-01, 1984-06-01`` setting,
-                        ``cellsize=cf.Y()`` will result in bounds of
-                        ``(1984-01-01, 1985-01-01), (1984-01-01,
-                        1985-01-01)`` (see `cf.Y` for details). Note that in
-                        this case each cell has the same bounds. This
-                        because ``cf.Y()`` is equivalent to ``cf.Y(month=1,
-                        day=1)`` and the closest 1st January to both
-                        coordinates is 1st January 1984.
-
-                {+data-like-scalar} TODO
-
-            flt: `float`, optional
-                When creating cells with sizes specified by the *cellsize*
-                parameter, define the fraction of the each cell which is
-                less its coordinate value. By default *flt* is 0.5, so that
-                each cell has its coordinate at it's centre. Ignored if
-                *cellsize* is not set.
+                *Parameter example:*
+                  For coordinates ``1, 2, 10``, setting ``bound=0.5``
+                  will result in bounds of ``(0.5, 1.5), (1.5, 2.5),
+                  (2.5, 17.5)``.
 
                 *Parameter example:*
-                    For coordinates ``1, 2, 10``, setting ``cellsize=1,
-                    flt=0.5`` will result in bounds of ``(0.5, 1.5), (1.5,
-                    2.5), (9.5, 10.5)``.
+                  Beware that if *bound* is too large (small), then
+                  some returned bounds values will by physically
+                  incorrect. For coordinates ``1, 2, 10``, the setting
+                  of ``bound=-2`` will result in bounds of ``(-2, 4),
+                  (4, 0), (0, 20)``.
+
+            cellsize: optional
+                Define the size of each cell that is created. Created
+                cells are allowed to overlap and do not have to be
+                contigious. The *cellsize* parameter may be one of:
+
+                * data_like scalar
+
+                  Defines the cell size, either in the same units as
+                  the coordinates or in the units provided. Note that
+                  in this case, the position of each coordinate within
+                  the its cell is determined by the *flt* parameter.
+
+            ..
```

Comment:
Quick question: what do you use these `..` for, I've always wondered (or assumed they were added by mistake, but I doubt so now)?

Reply:
Aha, that makes sense. Thanks for explaining.